app/views/users/edit.html.slim

```diff
@@ -0,0 +1,7 @@
+- provide(:create_btn_text, 'Edit User')
+- provide(:cancel_btn_text, 'Cancel')
+- provide(:path, users_path)
+h1.title
+  | Edit User
+.container
+  = render 'form', object: @user
```

Comment:
you are passing `@user` as `object` to the partial `form`, but are still calling `@user` in the partial
would you want the next?
```
= render 'form', user: @user
----------------
= simple_form_for user do |f|
```

Reply:
done